config/database.yml

```
@@ -19,7 +19,7 @@ default: &default
   encoding: unicode
   # For details on connection pooling, see Rails configuration guide
   # http://guides.rubyonrails.org/configuring.html#database-pooling
-  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  pool: <%= ENV['DATABASE_POOL_SIZE'] || ENV.fetch('RAILS_MAX_THREADS', 5).to_i %>
```

Maybe consistently use `.fetch` instead?

```suggestion
  pool: <%= ENV.fetch("DATABASE_POOL_SIZE") { ENV.fetch("RAILS_MAX_THREADS", 5).to_i } %>
```

(You'll notice our Rubocop config prefers double quotes, so we should be consistent in this regard too)

---

Whats the severity of deprecating one of these variables? I'm getting an error when using fetch on DATABASE_POOL_SIZE without a default. We can emulate existing behaviour by fetching with a default of nill 

`pool: <%= ENV.fetch("DATABASE_POOL_SIZE", nil) || ENV.fetch("RAILS_MAX_THREADS", 5).to_i %>`

Although having two config variables is confusing. Ideally this could be consolidated to

`pool: <%= ENV.fetch("DATABASE_POOL_SIZE", 5).to_i`

---

Oh! never mind I just noticed the suggestion has a return block to handle this.